daemon/src/oracle.rs

```diff
@@ -245,14 +244,26 @@ impl Actor {
         self.update_pending_attestations(ctx);
     }
 
-    async fn handle_new_attestation_fetched(&mut self, msg: NewAttestationFetched) {
+    async fn handle_new_attestation_fetched(&mut self, msg: NewAttestationFetched) -> Result<()> {
         let NewAttestationFetched { id, attestation } = msg;
 
         tracing::info!("Fetched new attestation for {id}");
 
-        let _: Result<(), xtra::Disconnected> =
-            self.attestation_channel.send_async_safe(attestation).await;
+        let mut conn = self.db.acquire().await?;
+
+        for id in db::load_all_cfd_ids(&mut conn).await? {
+            if let Err(err) = self
+                .executor
+                .execute(id, |cfd| cfd.decrypt_cet(&attestation))
+                .await
+            {
+                tracing::warn!("Failed to decrypt CET using attestation: {}", err)
```

Comment:
```suggestion
                tracing::warn!(order_id = %id, "Failed to decrypt CET using attestation: {}", err)
```

This should probably have the ID.